src/app/families/DeliveryStatus.ts

```diff
@@ -29,7 +35,12 @@ export class DeliveryStatus {
   static FailedBadAddress: DeliveryStatus = new DeliveryStatus(21, !use ? '' : use.language.notDeliveredBadAddress, true);
   static FailedNotHome: DeliveryStatus = new DeliveryStatus(23, !use ? '' : use.language.notDeliveredNotHome, true);
   static FailedDoNotWant: DeliveryStatus = new DeliveryStatus(24, !use ? '' : use.language.notDeliveredDontWant, true);
-  static FailedOther: DeliveryStatus = new DeliveryStatus(25, !use ? '' : use.language.notDeliveredOther, true);
+
+  static FailedNotReady: DeliveryStatus = new DeliveryStatus(26, !use ? '' : "התרומה עוד לא מוכנה לאיסוף", true);
+  static FailedAlreadyPickedUp: DeliveryStatus = new DeliveryStatus(27, !use ? '' : "התרומה כבר נמסרה", true);
+  static FailedTooFar: DeliveryStatus = new DeliveryStatus(28, !use ? '' : "רחוק לי", true);
+
+  static FailedOther: DeliveryStatus = new DeliveryStatus(29, !use ? '' : use.language.notDeliveredOther, true);
```

Comment:
You ignored my last comments ?

Reply:
you can't change the id of an existing status - without affecting existing data or migrating it.